actor.rb

```diff
@@ -0,0 +1,77 @@
+module Actor
+  module ClassMethods
+    def new(*args, &block)
```

Comment:
override the initialization of any class into which `Actor` is included and create a Proxy object that wraps the class being initialized